site/pages/about-us/q-and-a-slice/category/style.css

```diff
@@ -0,0 +1,51 @@
+@value badgerBlack, badgerRed, linesOnWhite from "../../../../css/_colors.css";
+@value mediumScreen, largeScreen from "../../../../css/_sizes.css";
+
+.category {
+  display: flex;
+  flex-wrap: wrap;
+  border-top: 2px solid badgerBlack;
+}
+
+.category__title {
+  composes: fontXS from "../../../../css/typography/_fonts.css";
+  composes: boldSansSerif from "../../../../css/typography/_fonts.css";
+  color: badgerRed;
+  text-transform: uppercase;
+  width: 100%;
+  padding-right: 10px;
+  padding-top: 20px;
+}
+
+.category__questionList {
+  width: 100%;
+}
+
+.category__element {
+  border-top: 2px solid linesOnWhite;
+}
+
+.category__element:first-child {
+  border-top: 0;
+}
+
+@media mediumScreen {
+  .category__title {
+    width: calc((100% + 20px) / 6);
+    padding-right: 20px;
+  }
+
+  .category__questionList {
+    width: calc((100% + 20px) * 5 / 6 - 20px);
+  }
```

Comment:
Just to breakdown that calculation:
A medium screen has six columns. The question list should take up 5 of those six columns. But one of those 5 columns doesn’t include the gutter. Gutters take up 20px on medium screen. So, we add on one gutter so we can take 5 / 6 of that. Then we remove the gutter at the end.